src/ng/components/body/body.html

```diff
@@ -1,6 +1,16 @@
-<tr ng-repeat="$row in $view.body.rows track by $index">
+<tr ng-repeat="$row in $view.body.rows track by $index"
+	 ng-class="{'selected': $selection.mode('row') && !$selection.checkbox && $selection.state($row)}"
+	 ng-click="$selection.mode('row') && !$selection.checkbox && $selection.toggle.execute($row)">
 	<td ng-repeat="$column in $view.body.columns track by $column.model.key"
-		 ng-class="[$column.model.key, $column.model.type]"
+		 ng-class="[
+			$column.model.key, 
+			$column.model.type, 
+		 	{
+				 'selected': $selection.mode('column') && $selection.state($column) ||
+								 $selection.mode('cell') && $selection.state($cell),
+			}]"
+		 ng-click="$selection.mode('column') && $selection.toggle.execute($column) || 
```

Comment:
This is bad idea to put ng-click on body(td|tr), look at body.core.js for editCell stuff.
Need to review ng-class in next versons too...we're triing to make for td.tr as less bindings as possible